editor-service/src/main/java/com/qreal/wmp/editor/controller/PaletteServletHandler.java

```diff
@@ -0,0 +1,109 @@
+package com.qreal.wmp.editor.controller;
+
+import com.qreal.wmp.editor.common.utils.AuthenticatedUser;
+import com.qreal.wmp.editor.database.exceptions.AbortedException;
+import com.qreal.wmp.editor.database.exceptions.ErrorConnectionException;
+import com.qreal.wmp.editor.database.exceptions.NotFoundException;
+import com.qreal.wmp.editor.database.palettes.client.PaletteService;
+import com.qreal.wmp.editor.database.palettes.model.Palette;
+import com.qreal.wmp.editor.database.palettes.model.PaletteView;
+import com.qreal.wmp.thrift.gen.PaletteServiceThrift;
+import com.qreal.wmp.thrift.gen.TPalette;
+import com.qreal.wmp.thrift.gen.TPaletteView;
+import org.apache.thrift.TException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Thrift PaletteRest controller.
+ * RPC functions for palette: createPalette
```

Comment:
+1
It is not stated formal (may be we should write it in wiki), but we got informal template for RPC interfaces. In doc should be listed all public functions of RPC interface (here 3).